src/Stryker.Core/Stryker.Core.UnitTest/TestRunners/VsTestRunnerPoolTests.cs

```diff
@@ -156,6 +179,26 @@ public void DetectTimeout()
             result.TimedOutTests.IsEmpty.ShouldBeFalse();
         }
 
+        [Fact]
+        public void RetryHangedSession()
```

Comment:
Okay sorry I cannot skip this anymore! _Hanged_ is used only for the method of execution, otherwise it's _hung_ :D

Reply:
yep, I did not remember the exact rule. Any useful synonym I could use instead (I am not fan of hung either)
:-)

Reply:
It could be _freeze_ probably, but I would live with whatever, just wanted to make a point :-P